src/directives/dnd.js

```diff
@@ -40,7 +40,14 @@ const DragAndDrop = Vue => {
         }
         e.target.classList.remove('drag-enter')
       }
+      const vm = this.vm
       this.handleDragEnd = e => {
+        console.log(this.vm)
+        // why this.vm is null inside the closure
+        // when you drag a tab over another and release ?
```

Comment:
Because at first called 'drop' event, where manipulations on tabs and vm lost after this